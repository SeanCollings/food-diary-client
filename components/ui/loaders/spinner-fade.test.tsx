import { render } from '@testing-library/react';
import { SpinnerFade } from './';

describe('<SpinnerFade>', () => {
  it('should render', () => {
    const { asFragment } = render(<SpinnerFade />);
    expect(asFragment()).toMatchSnapshot();
  });
});
