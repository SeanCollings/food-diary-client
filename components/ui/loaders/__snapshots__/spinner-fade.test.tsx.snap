// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SpinnerFade> should render 1`] = `
<DocumentFragment>
  <div
    class="sc-b46874d8-0 dsUvGF loader"
    color="var(--text)"
    size="45"
  />
</DocumentFragment>
`;
